### PR TITLE
Wish form fixes

### PR DESCRIPTION
**Pull request by stacysealky**
- Stripe added to package
- Fixed Wish Form Submission Pages (/users/profile and /users/choose)
- Deleted unused css
- Updated color scheme for consistency
- Added lines to WishCardRepository.js and /routes/wishCards.js (Patric's request)
- Added new helper modals to guide partner users better

### CODE REVIEW

File: client/views/profile.ejs [hunk 1]
```diff
@@ -1,115 +1,118 @@
...
+  </div>
+
+  <!-- profile content section-->
+  <header class="profile-header quick-font">
+    <div class="container justify-content-center">
+      <% if (!user.emailVerified) { %>
+      <p class="mt-3 quick-font notify">
+        <i class="fa fa-exclamation-triangle" aria-hidden="true"> </i>
+        Some features are disabled until email is verified.
+        Please verify your email.
```
Comment (Jacobjeevan) [inline client/views/profile.ejs:34]: If you change this line, you would need to modify the tests as well (they check for existence/non-existence of `Your email is unverified` (4 instances).

Example: https://github.com/donategifts/donategifts/blob/50356b0a3ab112e4090418e787a8b6cb39171a66/test/wishcards.test.js#L130

**Review by Jacobjeevan — CHANGES_REQUESTED** [review]

File: client/views/profile.ejs (hunk 1 above)
Comment (stacysealky) [reply client/views/profile.ejs:34]: oh gotcha, thanks for catching this!

Comment (Jacobjeevan) [reply client/views/profile.ejs:34]: Np.

There are few instances of `text.should.contain('Your email is unverified');` as well (should have made it clearer earlier, my bad).

File: server/db/repository/WishCardRepository.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ async function getViewableWishCards(showDonated) {
   try {
     const searchArray = [];
     searchArray.push({ status: 'published' });
+    // Patric told me to add this line -Stacy
+    searchArray.push({ status: 'draft' });
```
Comment (Enubia) [inline server/db/repository/WishCardRepository.js:27]:
```js
const searchArray = [{ status: 'published' }, { status: 'draft' }];
```
and remove the other 3 lines

**Review by Enubia — CHANGES_REQUESTED** [review]

File: server/db/repository/WishCardRepository.js (hunk 2 above)
Comment (stacysealky) [reply server/db/repository/WishCardRepository.js]: changed it

Comment (stacysealky) [reply server/db/repository/WishCardRepository.js]: @Enubia can you merge this once the checks pass?

**Review by Enubia — APPROVED** [review]

**Pull request merged**